Makefile

```diff
@@ -7,3 +7,6 @@ test:
 
 gen:
 	GEN_BINDING=1 cargo build
+
+set_variables_list.txt:
+	tesseract --print-parameters > $@
```

Comment:
I was thinking overnight that it might be better to generate the variables file as `src/variable.rs` and commit it.

That way, it would be easier to see the difference when the generator code is tweaked. And the generator script wouldn't necessarily have to be written in Rust.